src/react-components/home-root.js

```diff
@@ -80,9 +104,16 @@ class HomeRoot extends Component {
   render() {
     const mainContentClassNames = classNames({
       [styles.mainContent]: true,
-      [styles.noninteractive]: !!this.state.dialogType
+      [styles.noninteractive]: !!this.state.dialog
     });
-    const dialogTypes = InfoDialog.dialogTypes;
+
+    const onLinkClicked = trigger => {
```

Comment:
Not a huge deal, but I would move this out of render. I think I only notice these becasue in some previous projects it was a linting rule to not define funcitons in render. Not sure how much it matters in practice.

Reply:
OK, if you like that convention I will follow it.